### PR TITLE
Fix --shap and JIT warnings, remove TkInter dependency, add isort linter, move scripts

### PR DESCRIPTION
**Pull request by rotx-maxim**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ This software currently supports Ubuntu 18.04 LTS.
 The server version is sufficient, see https://ubuntu.com/download/server.
 *Note: The Windows Subsystem for Linux (WSL) currently does <u>not</u> support CUDA.*
 
-When going beyond simple tests, model training requires CUDA hardware acceleration (the network loader does not require CUDA).
+When going beyond simple models, model training does not work well without CUDA hardware acceleration (the network loader does not require CUDA, and very simple models can be trained on the CPU only).
```
Comment (aniktash) [inline README.md:58]: I suggest:"  ... and very simple models can still be trained on the CPU."

**Review by aniktash — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rotx-maxim) [reply README.md]: Thanks, I clarified this.

**Review by ermanok — APPROVED** [review]
Looks good.

File: ai8x.py [hunk 2]
```diff
@@ -149,45 +149,16 @@ def forward(self, x):  # pylint: disable=arguments-differ
...
-        self.quantized = quantized
-
-    def forward(self, x, s):  # pylint: disable=arguments-differ
+    def forward(self, x, s):  # pylint: disable=arguments-differ, no-self-use
         """Forward prop"""
-        if self.quantized:
-            return ScaleFunction.apply(x, s)
-        return s*x
+        if dev.simulate:
+            return FloorFunction.apply(x*s)
```
Comment (MaximGorkem) [inline ai8x.py:160]: Not having factor should create a mismatch between qat8 and post-quantization of qat8 for some cases. However, as we spoke, this is the best option until we find the issue.

**Review by MaximGorkem — APPROVED** [review]
Looks good.

**Review by MaximGorkem — APPROVED** [review]
Looks good.

File: ai8x.py (hunk 2 above)
Comment (rotx-maxim) [reply ai8x.py:160]: The best resolution would be to add a test case to test_qat.py.

**Pull request merged**